radio/apps/schedules/models.py

```diff
@@ -230,23 +220,22 @@ def save(self, *args, **kwargs):
 
     @classmethod
     def between(cls, after, before, exclude=None, live=False, schedule_board=None):
-        list_schedules = cls.objects.filter(
-            programme__start_date__lte=before, programme__end_date__isnull=True
-        ) | cls.objects.filter(
-            programme__start_date__lte=before, programme__end_date__gte=after
+        list_schedules = (
+            cls.objects.filter(programme__start_date__lte=before, programme__end_date__isnull=True) |
+            cls.objects.filter(programme__start_date__lte=before, programme__end_date__gte=after)
```

Comment:
Make code more readable. The important part is to apply multiple filters, so the code should not spread this across multiple indentation levels, but should reflect this by intuitive reading.